src/tiled/mapobjectmodel.h

```diff
@@ -97,7 +97,8 @@ class MapObjectModel : public QAbstractItemModel
     void setObjectRotation(MapObject *o, qreal rotation);
 
     void setObjectProperty(MapObject *o, MapObject::Property property, const QVariant &value);
-    void emitObjectsChanged(const QList<MapObject *> &objects, const QList<Columns> &columns = QList<Columns>());
+    void emitObjectsChanged(const QList<MapObject *> &objects, const QList<Column> &columns = QList<Column>());
+    void emitObjectsChanged(const QList<MapObject*> &objects, const Column column);
```

Comment:
Please leave out the `const` here, it makes no difference to the caller whether a parameter passed by value is declared const or not. Sorry to nitpick about it, but I don't want to do a commit just to remove it. :-)